.gitmodules

```diff
@@ -25,7 +25,7 @@
 	url = https://github.com/dmlc/cub
 [submodule "3rdparty/tvm"]
```

Comment:
If we need to upgrade to the latest commit of tvm (which depends on it own version of dmlc-core), it is necessary to upgrade the dmlc-core submodule in mxnet?

Reply:
a. any nnvm header that is included in MXNet .cc file will use the MXNet's dmlc-core
b. I'm not sure if the make of NNVM is done using its own dmlc-core - it is possible that the env is set up so all components use the same (MXNet's) dmlc-core. Otherwise you could end up with some binary incompatibilities (e.g. if dmlc-core changes some struct and it is passed around between NNVM and MXNet you don't want it to be interpreted differently).

Reply:
tvm now comes with a [notice](https://github.com/dmlc/tvm/blob/master/NOTICE) file, so we'd need to add that to mxnet's notice too per apache license.